core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -26,8 +26,10 @@ oppia.directive('simpleEditorSidebar', [
       controller: [
         '$scope', 'EditorModeService', 'SimpleEditorManagerService',
         'ScrollSyncService', 'QuestionIdService',
+        'SimpleEditorSidebarModeService',
         function($scope, EditorModeService, SimpleEditorManagerService,
-                 ScrollSyncService, QuestionIdService) {
+                 ScrollSyncService, QuestionIdService,
+                  SimpleEditorSidebarModeService) {
```

Comment:
I don't see any changes. It should be deindented by 1 space from where it is now to align with the line above.